src/components/CustomFormField.jsx

```diff
@@ -0,0 +1,19 @@
+import React from "react";
+import { Field } from "formik";
+
+const CustomFormField = props => {
+	const { component, name,  formikProps, onChange } = props;
+	const handleChange = changeEvent => {
+		const { name, value } = changeEvent.target;
+		formikProps.setFieldValue(name, value);
+		formikProps.setFieldValue('subRequestType', '');
```

Comment:
i see why this needs to be done, typically i would make this more generic so we could reuse. But mabye instead to keep this moving we can call this component

`RequestTypeField`?

Reply:
modified name to RequestTypeField

Reply:
Confirmed name changed